commands/deploy.go

```diff
@@ -697,3 +729,36 @@ func findMissingValues(signingDetails common.SigningDetails) string {
 	}
 	return strings.Join(missingValues, ",")
 }
+
+func getRepositoryNamePrefix(registry string) (string, error) {
+	parts := strings.Split(registry, "/")
+	if len(parts) < 3 {
+		return "", errors.New("registry must have a dockerhub owner or private registry and repo-name")
+	}
+	return strings.Join(parts[2:], "/"), nil
+}
+
+// Checks if the repostitory exists in the compartment
+func doesRepositoryExistInCompartment(repositoryName string, compartmentID string, artifactsClient artifacts.ArtifactsClient) (bool, error) {
+	response, err := artifactsClient.ListContainerRepositories(context.Background(), artifacts.ListContainerRepositoriesRequest{
+		CompartmentId: &compartmentID,
+		DisplayName:   &repositoryName})
+	if err != nil {
+		return false, fmt.Errorf("failed to lookup Container Repository due to %w", err)
+	}
+	if *response.RepositoryCount == 1 {
```

Comment:
I'm not sure about this suggestion, but it seems that if the count is bigger than one it exists.
```suggestion
	if *response.RepositoryCount >= 1 {
```